packages/service-utils/src/core/authorize/index.ts

```diff
@@ -34,31 +35,31 @@ export async function authorize(
   serviceConfig: CoreServiceConfig,
   cacheOptions?: CacheOptions,
 ): Promise<AuthorizationResult> {
-  const { clientId, targetAddress, secretKeyHash } = authData;
+  const { clientId, targetAddress, secretKeyHash, authToken } = authData;
   const { enforceAuth } = serviceConfig;
 
   // BACKWARDS COMPAT: if auth not enforced and
   //                   we don't have auth credentials bypass
-  if (!enforceAuth && !clientId && !secretKeyHash) {
+  if (!enforceAuth && !clientId && !secretKeyHash && !authToken) {
     return {
       authorized: true,
       apiKeyMeta: null,
     };
   }
 
-  // if we don't have a client id at this point we can't authorize
-  if (!clientId) {
```

Comment:
this file needs another look in its entirety now that we have auth token to consider